app/controllers/galaxies_controller.rb

```diff
@@ -6,10 +6,26 @@ def show
   end
 
   def new
-    @owner= User.find(params[:user_id])
+    @owner = User.find(params[:user_id])
     @galaxy = Galaxy.new
   end
 
+  def create
+    @galaxy = Galaxy.new(galaxy_params)
+    @owner = User.find(params[:user_id])
+    @galaxy.owner = @owner
+    @galaxy.save
+
+    redirect_to galaxies_show_path(@galaxy)
```

Comment:
- [ ] We should consider adding an IF statement for the redirect and save actions.